### PR TITLE
Current location

### PR DESCRIPTION
**Pull request by nsantos16**
### Description
As a user I should be able to allow the browser to detect my location, so the map can be positioned on it and find people with similar interests.

### Preview
ioscurrent

### CODE REVIEW

File: src/components/common/TextInputExtend/index.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ class TextInputExtend extends React.Component {
     return <TextInput
       {...rest}
       onChange={this.handleChange}
-      style={placeholder ? [style, placeholderStyle] : style}
+      style={[style, placeholder ? placeholderStyle : { color: '#000' }]}
```
Comment (missx) [inline src/components/common/TextInputExtend/index.js]: Can you store that color as a constant too?

File: src/components/main/Map/index.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      error: null
+    };
+  }
+
+  getPosition() {
+    navigator.geolocation.getCurrentPosition((position) => {
+      this.setState({
+        region: {
+          latitude: position.coords.latitude,
+          longitude: position.coords.longitude,
```
Comment (missx) [inline src/components/main/Map/index.js]: You can deconstruct `position.coords` first and only add `latitude` and `longitude` to the region object.

File: src/components/main/Map/index.js [hunk 3]
```diff
@@ -1,16 +1,52 @@
...
+          latitudeDelta: 0.003,
```
Comment (felipegarcia92) [inline src/components/main/Map/index.js]: Can you declare this `0.003` value as a constant? Something like `DEFAULT_DELTA` is fine.

**Comment by agustito37** [comment]
Hi @nsantos16 If I'm not wrong this PR has a dependency with the PR https://github.com/nsantos16/targetMvd/pull/3, in these cases you should put this PR 'on hold' with a label so we don't correct the same things twice.

File: src/components/main/Map/index.js [hunk 4]
```diff
@@ -17,7 +17,7 @@ class Map extends React.Component {
 
   getPosition() {
     navigator.geolocation.getCurrentPosition((position) => {
```
Comment (agustito37) [inline src/components/main/Map/index.js]: Alo! You can destructure like this `.getCurrentPosition({ coords: { latitude, longitude } })`

**Pull request merged**